.github/workflows/vale.yaml

```diff
@@ -0,0 +1,17 @@
+name: vale
+on: [pull_request]
+
+jobs:
+  vale:
+    name: runner / vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
```

Comment:
Instead of using the `reviewdog` branch, should we use the `v2` or `v2.1.0` tag?

Reply:
Changed to `v2`